rllib/offline/estimators/fqe_torch_model.py

```diff
@@ -113,14 +115,18 @@ def train_q(self, batch: SampleBatch) -> TensorType:
             A list of losses for each training iteration
         """
         losses = []
+        if self.minibatch_size is None:
+            minibatch_size = batch.count
         for _ in range(self.n_iters):
             minibatch_losses = []
             batch.shuffle()
-            for idx in range(0, batch.count, self.batch_size):
-                minibatch = batch[idx : idx + self.batch_size]
+            for idx in range(0, batch.count, minibatch_size):
+                minibatch = batch[idx : idx + minibatch_size]
                 obs = torch.tensor(minibatch[SampleBatch.OBS], device=self.device)
                 actions = torch.tensor(
-                    minibatch[SampleBatch.ACTIONS], device=self.device
+                    minibatch[SampleBatch.ACTIONS],
+                    device=self.device,
+                    dtype=int,
```

Comment:
why do you need to enforce `int` type? Actions should already be int.

Reply:
I believe our minibatch promotes them to `np.float32` somewhere in `convert_to_numpy`? It crashes if I remove that line.

Reply:
🤔 use torch.int then

Reply:
Done